libraries/EmbeddedChessPieces/src/ECPMovement/ECPMovement.cpp

```diff
@@ -0,0 +1,26 @@
+#include "ECPMovement.h"
+
+ECPMovement::ECPMovement(Dezibot &d, uint mc = 3900) : dezibot(d), ecpColorDetection(ECPColorDetection(d)), movementCalibration(mc) {};
+
+void ECPMovement::moveForward(int timeMovement, int timeBreak) {
+    dezibot.motion.move(0, movementCalibration);
+    delay(timeMovement);
+    dezibot.motion.stop();
+    delay(timeBreak);
+};
+
+void ECPMovement::moveToNextField() {
+    bool startedOnWhite = ecpColorDetection.isWhiteField();
+    bool currentlyOnWhite = startedOnWhite;
```

Comment:
```suggestion
    const bool hasStartedOnWhite = ecpColorDetection.isWhiteField();
    bool isCurrentlyOnWhite = hasStartedOnWhite;
```

`startedOnWhite` can be `const`.

Also, please use `is` / `has` / ... prefixes for boolean values as this is recommended for clean(er) code. (Needs to be updated in the function as well.)